### PR TITLE
Fixed typo in doc

### PR DESCRIPTION
**Pull request by mganter**
**How to categorize this PR?**

/area documentation
/kind bug
/priority 3

**What this PR does / why we need it**:

Fixes typo in tutorial

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

component descriptor now defines `repositoryContexts` instead of `repositoryContext`
https://gardener.github.io/component-spec/component-descriptor-v2.html#component_repositoryContexts

Currently `landscaper-cli blueprint render` leads to 
`unable to decode component descriptor: component: repositoryContexts is required`

**Release note**:

```other operator
NONE
```

### CODE REVIEW

**Comment by gardener-robot-ci-1** [comment]
Thank you @mganter for your contribution. Before I can start building your PR, a member of the organization must set the required label(s) {'reviewed/ok-to-test'}. Once started, you can check the build status in the PR checks section below.

File: docs/tutorials/01-create-simple-blueprint.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ component:
   sources: []
   componentReferences: []
 
-  respositoryContext:
+  respositoryContexts:
```
Comment (reshnm) [inline docs/tutorials/01-create-simple-blueprint.md:101]:
```suggestion
  repositoryContexts:
```

**Review by reshnm — CHANGES_REQUESTED** [review]

File: docs/tutorials/01-create-simple-blueprint.md (hunk 1 above)
Comment (mganter) [reply docs/tutorials/01-create-simple-blueprint.md]: thx :D would have introduced another typo

**Review by schrodit — APPROVED** [review]

**Review by reshnm — APPROVED** [review]
/lgtm

**Pull request merged**